agent/eventhandler/handler_test.go

```diff
@@ -203,165 +102,99 @@ func TestSendsEvents(t *testing.T) {
 	}
```

Comment:
We don't need it; `AddContainerEvent` already creates goroutines inside and has async behavior.